src/components/cities-index.js

```diff
@@ -0,0 +1,62 @@
+import { useSelector } from "react-redux";
+import { fetchCities } from "../actions";
+import { Sparklines, SparklinesLine, SparklinesReferenceLine } from 'react-sparklines';
+import SearchValue from "./search-form.js";
+import _ from 'lodash';
+
+const CitiesIndex = () => {
+  const cities = useSelector((state) => state.cities);
+
+  function renderCities() {
+    if (!_.isEmpty(cities)) {
+      let resultTemp = cities.stats.map(t => t.temperature);
+      let resultPressure = cities.stats.map(t => t.pressure);
+      let resultHumidity = cities.stats.map(t => t.humidity);
+
+      return (
+        <tr >
```

Comment:
It might make sense to create another component for this just because it's a bit crowded to read here